projects/demo-maps/src/app/route-components/route-example-layers/route-map.component.ts

```diff
@@ -308,10 +308,10 @@ export class RouteMapComponent implements OnInit {
   }
 
   addOverlays() {
-    const layerOnToppOfAll = new BlueMarbleTile()
-    const openSeaMapOnTopp = new OpenSeaMap();
-    this.layersSvc.addLayer(layerOnToppOfAll, 'Overlays');
-    this.layersSvc.addLayer(openSeaMapOnTopp, 'Overlays');
+    const layerOnTopOfAll = new BlueMarbleTile({crossOrigin: 'anonymous'});
```

Comment:
Maybe try to create an example why adding crossOrigin is useful.

Reply:
Adding crossOrigin is useful when we want to do something with a tainted canvas, like exporting it to PDF. When PDF-capabilities have been integrated into the frontend-libraries, I'll add an example to the demo-maps showcasing the importance of crossOrigin. Until then I've been asked to keep those two changes (crossOrigin capabilities and PDF-capabilities) separate.